CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -732,7 +732,7 @@ msgstr ""
 
 #. 3.2->settings-schema.json->enable-animation->description
 #. 2.6->settings-schema.json->enable-animation->description
-msgid "Use menu animations"
+msgid "Animate Menu opening and closing"
```

Comment:
Sounds better. Should be also changed upstream.
Maybe
Animate opening and closing the menu?